### PR TITLE
Simplify field reordering, preserve event handlers on unchanged DOM.

### PR DESCRIPTION
**Pull request by jtwalters**
A `.detach()` polyfill was needed for jQuery 1.3 backward compatibility.

### CODE REVIEW

**Comment by iamEAP** [comment]
FYI, this is in for `1.0.x` here: https://github.com/tableau-mkt/jquery.addressfield/commit/b34e1e8b8e51347e6e17c3020ccfdaa344f8ee32
